tools/java/cpp-build/test/com/google/i18n/phonenumbers/BuildMetadataCppFromXmlTest.java

```diff
@@ -175,11 +176,11 @@ public MockedCommand(String expectedInputFilePath, boolean expectedLiteMetadata,
       assertEquals(expectedVariant, variant);
       return sourceOut;
     }
-    String capturedHeaderFile() {
-      return new String(headerOut.toByteArray(), UTF_8);
+    String capturedHeaderFile() throws UnsupportedEncodingException {
+      return new String(headerOut.toByteArray(), "UTF-8");
```

Comment:
Okay with the change, but in that case, you need to remove line 144 since it's not used anymore.